### ISSUE TITLE
Finetuning: Big boi hyperparameter seach

### ISSUE THREAD

**Issue by sorenmulli** [peleiden/daluke#53]

**Comment by sorenmulli** [comment]
Idé:
epochs: 5, 15
lr: 2e-5, 5e-5
batch-size: 16, 32
weight-decay: 0.01, 0.05
dropout: 0.05, 0.1, 0.6
loss-weight: yes/no

Giver 96 fine-tuninger svarende til cirka 48 timers træning. Måske lige i overkanten - hvad tænker du, vi skal ændre, @asgerius?
Skal så også lige finde en måde, at køre det på - der skal nok lige være et script mere.

**Comment by sorenmulli** [comment]
thoughts, @asgerius ?

**Comment by asgerius** [comment]
Learning rate is maybe not intereseting enough? I would also like to see 10 epochs. Batch size 8 might also be very nice.
Dropout should be 0, 0.1, and 0.2 or 0.3 maybe? 0.6 seems too high

**Comment by asgerius** [comment]
Also can we even do batch size of 32 without grad accumulate? I think 24 might be the limit if we want a "pretty" number

**Comment by sorenmulli** [comment]
Agree with lr - should it just be fixed at 2e-5?
Lets do batch size of 16 and 24. Dropout could very well bee 0, 0.1 and 0.3.
Why 10 epochs doeee? When we use the best dev performance, we don't need that much diversity in that param

**Comment by sorenmulli** [comment]
Basse, så er det nu, der skal ændres på https://github.com/peleiden/daluke/blob/master/configs/param-lists.json eller ties for stedse, @asgerius

**Comment by asgerius** [comment]
Nyest foteslåede af hver parameter er good. Bare sæt 15 epoker på hver og ikke undersøg den parameter, da vi jo tager bedste

**Comment by sorenmulli** [comment]
Altså med nyest foreslåede mener du enighed med json'en derinde - den er ikke helt ens med hvad vi har talt om her, da vi har opdaget nye ting, da vi legede med baseline

**Comment by asgerius** [comment]
Den ser fin ud - ud over naturligvis epokerne. Måske vægthenfald er lidt dog? Men de har self heller ikke så lang tid at decaye på, så lad os bare sige, at det er fint

**Comment by sorenmulli** [comment]
Vi har sat den en sjat op fra 0.1, som vi altid kørte med, så tænker, det ville lidt wildo at sætte den højere op :S Fixer epoker
